sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-kernel/coreos-kernel-6.6.76.ebuild

```diff
@@ -61,9 +61,8 @@ src_prepare() {
 	# Pull in the config and public module signing key
 	KV_OUT_DIR="${SYSROOT%/}/lib/modules/${COREOS_SOURCE_NAME#linux-}/build"
 	cp -v "${KV_OUT_DIR}/.config" build/ || die
+
 	local sig_key="$(getconfig MODULE_SIG_KEY)"
```

Comment:
I said you could call `get_sig_key` here, but now it's not actually used below, so this line should go. I'm not sure whether you need something in its place. I guess it builds the modules first and will reuse the config from that, so you don't need to call `envsubst` again.